app/src/main/java/org/coepi/android/ui/symptoms/SymptomsViewModel.kt

```diff
@@ -75,7 +78,7 @@ class SymptomsViewModel (
             .observeOn(mainThread())
             .subscribe{
                 uiNotifier.notify(it)
-                navigation.navigate(Back)
+                navigation.navigate(ToDestination(ThanksFragmentDirections.actionGlobalThanksFragment()))
```

Comment:
Minor thing (but very quick to do), you can use static import to make this a little less verbose. If you press alt+enter on `actionGlobalThanksFragment` AS offers to add it.